pgextwlist.c

```diff
@@ -360,9 +360,29 @@ extwlist_ProcessUtility(PROCESS_UTILITY_PROTO_ARGS)
 				 */
 				if (all_in_whitelist)
 				{
-					call_ProcessUtility(PROCESS_UTILITY_ARGS,
-										NULL, NULL, NULL, NULL, NULL);
-					return;
+					char *current_user = get_current_user_name();
+					char *database_owner = get_current_database_owner_name();
+					
+					if(current_user == NULL || database_owner == NULL)
+					{
+						ereport(ERROR,
+							(errcode(ERRCODE_INTERNAL_ERROR),
+							errmsg("Invalid current user (%s) or database owner (%s)", current_user, database_owner)));
```

Comment:
I'd say we don't print the DB owner name in case it's not null so that it's not logged plaintext style

Reply:
should that be a secret?:)

Reply:
this logic lets you send null as the current user and potentially would log the DB owner user. Maybe I'm being overly cautious but I think we should avoid it.